src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023 the ThorVG project. All rights reserved.
+
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#ifndef _TVG_WG_GEOMETRY_H_
+#define _TVG_WG_GEOMETRY_H_
+
+#include <cassert>
+#include <cmath>
+#include "tvgCommon.h"
+#include "tvgArray.h"
+
+class WgPoint
+{
+public:
+    float x;
+    float y;
+
+public:
+    WgPoint() {}
+    WgPoint(float x, float y): x(x), y(y) {}
+    WgPoint(const Point& p): x(p.x), y(p.y) {}
+
+    inline WgPoint operator + (const WgPoint& p) const { return { x + p.x, y + p.y }; }
```

Comment:
Suggest simpler code: --inline
(these inline specificier are unnecessary,  A header itself is inline)

Reply:
fixed